frontend/src/app/testResults/constants.ts

```diff
@@ -5,6 +5,7 @@ export enum MULTIPLEX_DISEASES {
   RSV = "RSV",
   HIV = "HIV",
   SYPHILIS = "Syphilis",
+  FLU_A_AND_B = "Flu A and B",
```

Comment:
I think adding this here and adding "Flu A and B" to the results page condition filter may have broken filtering the results and then downloading it. 

So up to this point everything looks good:
<img width="1277" alt="Screenshot 2024-07-24 at 12 34 14" src="https://github.com/user-attachments/assets/dad2b277-56bf-44db-a808-fbe9b2de895d">

But when you download the file and actually open it the condition filter didn't actually work

Reply:
uh oh I don't think this works on `main` either. I will see if the fix is something I can pull into this PR or if it's better attacked separately

Reply:
OOO oops -- sorry I didn't check `main` before commenting! I can create a separate bug ticket!

Reply:
Created a bug for it here: https://github.com/CDCgov/prime-simplereport/issues/7948

Reply:
thank you! and good catch. looks like we forgot to update the Download results functionality when we added Condition and Result filtering to the result page

Reply:
update: won't pull that fix into this PR since I think it's out of scope and the fix is in a different part of the code